src/code42cli/logger/__init__.py

```diff
@@ -5,13 +5,13 @@
 from logging.handlers import RotatingFileHandler
 from threading import Lock
 
-from c42eventextractor.logging.formatters import FileEventDictToCEFFormatter
-from c42eventextractor.logging.formatters import FileEventDictToJSONFormatter
-from c42eventextractor.logging.formatters import FileEventDictToRawJSONFormatter
-from c42eventextractor.logging.handlers import NoPrioritySysLogHandlerWrapper
 from click.exceptions import ClickException
 
 from code42cli.cmds.search.enums import FileEventsOutputFormat
+from code42cli.logger.formatters import FileEventDictToCEFFormatter
```

Comment:
just moved these form the extractor to here so now the only thing that remains in the extractor is the extractor itself, so one step closer to being able to delete that project